src/mixins/helpers.js

```diff
@@ -108,14 +107,14 @@ var helpers = {
         if (this.props.afterChange) {
           this.props.afterChange(currentSlide);
         }
-        ReactTransitionEvents.removeEndEventListener(this.refs.track.getDOMNode().children[currentSlide], callback);
+        delete this.animationEndCallback;
```

Comment:
I delete the reference to the animation timer as it is no longer necessary.
There is no need to to a clearTimeout as the timeout has already ended.